### DISCUSSION TITLE
Ease of Use Improvements: better help / support folder inputs

### DISCUSSION THREAD

**Discussion by turnkit** [complexlogic/rsgain#123] [Ideas]
I've folders with thousands of meeting recordings and don't want to use the album but individual track setting.  Please add a flag for easy mode to ignore album option.

Also, please add more flexible help info so that all the help is shown with --help or -h.   App seems picky to find -h custom when I am just starting trying to understand what the options are.

(Also, any idea why VLC doesn't recognize the REPLAYGAIN setting even when it's turned on in preferences? Is this a new bug with VLC?)

**Comment by complexlogic** [comment]
> Please add a flag for easy mode to ignore album option.

This is possible with the scan presets feature. Pass `-p no_album` to the program in Easy Mode

> Also, please add more flexible help info so that all the help is shown with --help or -h. App seems picky to find -h custom when I am just starting trying to understand what the options are.

The options are different depending on what mode the program is operating in. For example, there is a `-m` option in both Custom Mode and Easy Mode, but they have different meanings. That's why there are separate help screens.

> (Also, any idea why VLC doesn't recognize the REPLAYGAIN setting even when it's turned on in preferences? Is this a new bug with VLC?)

What file format are you using and how are you determining that the tags are not being recognized?
